One Identity/one-identity-sps/ingest/parser.yml

```
@@ -0,0 +1,56 @@
+name: one-identity
+ignored_values: []
+pipeline:
+  - name: parse_cef
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: set_fields
+    description: ''
+    filter: ''
+    external: null
+stages:
+  set_fields:
+    actions:
+      - set:
+          '@timestamp': '{{parse_date.datetime}}'
```

As the `parse_date` stage was withdrawn, as requested, the current line is no more valid.
It seems that we can replace it with the `start` field from the original message.
```suggestion
          '@timestamp': '{{parse_cef.start | to_rfc3339}}'
```